docs/user-guide/command-line-interface.md

```diff
@@ -69,7 +69,9 @@ Miscellaneous:
   --debug                    Output debugging information
   -h, --help                 Show help
   -v, --version              Outputs the version number
-  ```
+  --no-inline-config         Prevent comments from changing eslint rules -
```

Comment:
`inlineConfig` is `true` by default: https://github.com/eslint/eslint/pull/4186/files#diff-763aa8deb604d725a74f8304903c1422R205